central/metrics/init.go

```diff
@@ -11,10 +12,16 @@ var (
 
 func init() {
 	// general
+
+	if !env.PostgresDatastoreEnabled.BooleanSetting() {
+		prometheus.MustRegister(
+			boltOperationHistogramVec,
+			rocksDBOperationHistogramVec,
+			dackboxOperationHistogramVec,
+		)
+	}
+
 	prometheus.MustRegister(
-		boltOperationHistogramVec,
-		rocksDBOperationHistogramVec,
-		dackboxOperationHistogramVec,
 		graphQLOperationHistogramVec,
 		graphQLQueryHistogramVec,
 		indexOperationHistogramVec,
```

Comment:
are we using this one for Postgres?

Reply:
Yeah, technically search queries go through it

Reply:
Post RocksDB, let's move this to like a postgres_search_op_duration or something like that

Reply:
sounds good.